backup_and_restore/backing-up-etcd.adoc

```diff
@@ -0,0 +1,30 @@
+[id="backup-etcd"]
+= Backing up etcd
+include::modules/common-attributes.adoc[]
+:context: backup-etcd
+
+toc::[]
+
+etcd is the key value store for all object definitions, as well as the
+persistent master state. Other components watch for changes, then bring
+themselves into the desired state.
+
+It is recommended to back up your cluster's etcd data regularly and store in a
+secure location ideally outside the {product-title} environment. Do not take an
+etcd backup before the first certificate rotation completes, which occurs 24
+hours after installation, otherwise the backup will contain expired
+certificates. It is also recommended to take etcd backups during non-peak usage
+hours, as it is a blocking action.
+
+Once you have an etcd backup, you can xref:../backup_and_restore/disaster_recovery/scenario-1-infra-recovery.adoc#dr-infrastructure-recovery[recover from lost master hosts]
+and xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore back to a previous cluster state].
+
+// TODO: Removed "and where the etcdctl tool is installed." - okay, or is that
+// necessary??
+// Does it have to be a master host?
```

Comment:
I think this is important, assuming cluster is running backup should be run inside a container. I will talk to @retroflexer  about adding this.

Reply:
@hexfusion So do I need to update this statement to say "on any master host that has connectivity..."? Or leave it as is?

Reply:
etcdctl needs to run inside the container (part of the pod), and not on master host (unless we downloaded etcdctl, as we do part of recovery script). I can't see this section anymore.

Reply:
@retroflexer Sure, but the script takes care of all that, running the etcdctl commands, correct? And the etcd-snapshot-backup.sh script is run from a master host according to the procedure [1], so I was just wondering if this statement:

`You can perform the etcd data backup process on any host that has connectivity to the etcd cluster, where the proper certificates are provided.`

Should clarify to say:

`You can perform the etcd data backup process on any master host that has connectivity to the etcd cluster, where the proper certificates are provided.`

Because the master host is where the etcd backup process is initiated from. Or could it be any host? 

[1] https://bz-1744569--ocpdocs.netlify.com/openshift-enterprise/latest/backup_and_restore/backing-up-etcd.html#backing-up-etcd-data_backup-etcd